Changes.txt

```diff
@@ -975,3 +975,5 @@ Beta 1  - 21/jan/2005
 - Fixed: Remove method
 
 - Fixed: Windsor: Proxy for components with (service != impl)
+
+- Changed: Performance Counter InstanceLifetime
```

Comment:
Thanks for updating the changelog, however it is sorted with the newest releases at the top. Could you move this to the first line in the file, we add the release line once we ship.